package-lock.json

```diff
@@ -5486,7 +5486,7 @@
       "dependencies": {
         "bl": {
           "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
+          "resolved": "http://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
```

Comment:
Please revert this change

Reply:
changes reverted

Reply:
… now all the blog files are back. 😲

Reply:
@fhemberger  my bad :( during the revert process. 
anyway, I removed them and i removed all the files that are not translated yet.
thank you for your patience